carma_wm/include/carma_wm/collision_detection.h

```diff
@@ -55,20 +55,33 @@ namespace carma_wm {
             public:
                 /*! \brief Main collision detection function to be called when needed to check for collision detection of the vehicle with 
                 * the current trajectory plan and the current world objects
+                * \param size The size of the host vehicle defined in meters
+                * \param rwol The list of Roadway Obstacle
+                * \param tp The TrajectoryPlan of the host vehicle
+                * \param size The size of the host vehicle defined in meters
+                * \param veloctiy of the host vehicle
+                * \param target_time amount of unit of time in future to look for collision
```

Comment:
What are the units? The argument is an int which makes me think they are not seconds. If they are miliseconds or something then an unsigned int would be a better type since negative time does not make sense.